src/main/java/com/indeed/jiraactions/api/response/issue/Issue.java

```diff
@@ -19,20 +28,61 @@
     public Field fields;
     public ChangeLog changelog;
 
+    // Multivalued rich object fields are treated different in Jira changelog histories than
+    //  multivalued primitive fields.
+    private static Set<String> MULTIVALUED_RICH_FIELDS = ImmutableSet.of(
```

Comment:
I think we can derive this list from from /api/2/field/. A good future enhancement. (The key is the "items": "string" versus some other thing.)